atomic_reactor/util.py

```diff
@@ -590,6 +591,21 @@ def is_scratch_build():
         raise
 
 
+def is_isolated_build():
+    build_json = get_build_json()
+    try:
+        return build_json['metadata']['labels'].get('isolated', False)
+    except KeyError:
+        logger.error('metadata.labels not found in build json')
+        raise
+
+
+def get_orchestrator_platforms(workflow):
+    for plugin in workflow.buildstep_plugins_conf or []:
+        if plugin['name'] == PLUGIN_BUILD_ORCHESTRATE_KEY:
+            return plugin['args']['platforms']
```

Comment:
I don't think it needs to derail this PR but... could this be in build metadata? It does not feel like a plugin arg is the place this should be coming from if it affects anything outside that plugin.

Reply:
Quick and dirty fix, but yes, it's inelegant. One of the reasons I pushed it into its own function is so it would be encapsulated and could be changed to something sensible later.